src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.duke.commands;
+
+import seedu.duke.data.Item;
+import seedu.duke.data.ShoppingList;
+
+public class AddCommand extends Command {
+
+    public static final String COMMAND_WORD = "ADD";
+    public static final String SUCCESS_ACK = "Added this item: " + "%s";
+    private Item Add;
+
+    public AddCommand(String description, double price){
+
+        this.Add = new Item(description, price);
+    }
+
+    @Override
+    public CommandResult execute()
+    {
+        Item.add(Add);
```

Comment:
Here can call a ShoppingList.add method instead of Item.add, then in ShoppingList call items.add

Reply:
okay! thank you ^^